lib/logging.properties

```diff
@@ -25,6 +25,7 @@ org.jitsi.jibri.selenium.util.BrowserFileHandler.count = 10
 org.jitsi.jibri.selenium.util.BrowserFileHandler.limit = 10000000
 
 org.jitsi.level = FINE
+org.jitsi.jibri.config.level = INFO
```

Comment:
Is this the right package name? I see only `JibriConfig` there and it doesn't even have its own logger

Reply:
The name we use for the metaconfig logger is manually set to that: https://github.com/jitsi/jibri/blob/master/src/main/kotlin/org/jitsi/jibri/Main.kt#L218